docs/project_MDMA/about.md

```diff
@@ -0,0 +1,56 @@
+---
+id: about
+title: Material Donor Mutual Assist
+custom_edit_url: null
+---
+
+## Overview
+
+The Material Donor Mutual Assist project by BWorks is designed to streamline the process of managing donations, providing a transparent view into the journey of each donated item. This initiative allows donors to track their contributions from the moment they are donated to their ultimate use, offering them a detailed view of how their items are making a difference. By keeping donors informed about the status and use of their donations, the project fosters a sense of trust and community, encouraging continued support and involvement. The system simplifies the administrative tasks involved in updating and managing donations, making it easier to attract and retain donors who are motivated by seeing the tangible impact of their contributions.
+
+### Information
+
+- **Source Code:** [https://github.com/slu-csci-5030/material-donor-mutual-assist/](https://github.com/slu-csci-5030/material-donor-mutual-assist/) [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/slu-csci-5030/material-donor-mutual-assist/)
```

Comment:
This repo will not be available to outsiders. We should clone this into the OSS organization. I'll merge it for now, but let's follow up with an edit once that's done.